### PR TITLE
integrate the force flag from cli with the generate_upgrade_plan

### PR DESCRIPTION
**Pull request by gabrielcocenza**
- update unit tests

Should be merged after #239

### CODE REVIEW

**Review by agileshaw — APPROVED** [review]
LGTM

File: cou/apps/base.py [hunk 1]
```diff
@@ -411,19 +411,19 @@ def post_upgrade_steps(self, target: OpenStackRelease) -> list[PostUpgradeStep]:
     def generate_upgrade_plan(
         self,
         target: OpenStackRelease,
+        force: bool,
```
Comment (rgildein) [inline cou/apps/base.py:414]: Can we use default value to `False`?

**Review by rgildein — APPROVED** [review]
One question, but that's not blocking one.

File: cou/apps/base.py (hunk 1 above)
Comment (gabrielcocenza) [reply cou/apps/base.py:414]: We can, but I prefer not to so it forces passing the CLI parameter for it.

Comment (rgildein) [reply cou/apps/base.py:414]: Why? What is the benefit of such approach?

Comment (rgildein) [reply cou/apps/base.py:414]: Discussed and we will change it latter, if we do not forget.

**Pull request merged**